spec/factories/in_person_enrollments.rb

```diff
@@ -14,6 +16,28 @@
       after :build do |enrollment|
         enrollment.status = :pending
         enrollment.enrollment_code = Faker::Number.number(digits: 16)
+        enrollment.enrollment_established_at = Time.zone.now
+        enrollment.status_updated_at = Time.zone.now
+      end
+    end
+
+    trait :expired do
+      after :build do |enrollment|
+        enrollment.status = :expired
+        enrollment.enrollment_code = Faker::Number.number(digits: 16)
+        enrollment.enrollment_established_at = Time.zone.now
+        enrollment.status_check_attempted_at = Time.zone.now
+        enrollment.status_updated_at = Time.zone.now
+      end
+    end
+
+    trait :failed do
+      after :build do |enrollment|
+        enrollment.status = :failed
+        enrollment.enrollment_code = Faker::Number.number(digits: 16)
+        enrollment.enrollment_established_at = Time.zone.now
+        enrollment.status_check_attempted_at = Time.zone.now
+        enrollment.status_updated_at = Time.zone.now
       end
```

Comment:
do these need to be `after` blocks? could the more direct syntax work too? (my thinking is that `after` might make for confusing results if we try to instantiate the factory and override speiifc attributes but the after block changes the ordering)

```suggestion
      status { :failed }
      enrollment_code { Faker::Number.number(digits: 16) }
      enrollment_established_at { Time.zone.now }
      status_check_attempted_at { Time.zone.now }
      status_updated_at { Time.zone.now }
```